### DISCUSSION TITLE
Where can I find apiman API Rest Docs

### DISCUSSION THREAD

**Discussion by spsvon** [apiman/apiman#2456] [Help]
Hello,

I've been finding into Apiman web Documentation about apiman API Rest Documentation (for manager/gateway). 

I've found  https://www.apiman.io/latest/api-manager-restdocs.html into main Documentation, but It throws a not found error.

Are there any link to apiman API definition (like swagger pager or something similar)?

I want to automate the creation of apiman clients and creation of contracts via API Rest Calls, but I need to know what parameters are mandatory to access each operation/method.

Thanks,
Sergio

**Comment by volkflo** [comment]
In the Apiman UI you can directly click your username and open the `REST Documentation` of the manager.
URL: `https://your-host/apimanui/api-manager/def`

**Comment by msavy** [comment]
Old qmino docs (I've just bumped up to a version of the plugin that supports Java 11):
   https://www.apiman.io/api-docs/apiman-manager
   https://www.apiman.io/api-docs/apiman-gateway

Work in progress for OpenAPI v3 via Redocly (I'm still changing over annotations, etc; temporary location):
    https://www.apiman.io/rest-api-docs.html

**Reply by msavy** [reply]
Oh, looks like the qmino docs aren't rendering properly on GitHub Pages for some reason. It works when running locally 🤔.

I'll try to fix that today, but hopefully the OpenAPIv3 ones will be OK for now even though they aren't perfect.

**Reply by msavy** [reply]
Okay, I figured it out. The QMino code is not compatible with Javascript minification. I'll exclude those directories from the build pipeline and it should start working.

**Reply by msavy** [reply]
Okay, pipeline has been updated and run again @spsvon — should be working now.

**Reply by msavy** [reply]
I've also added links to the Apiman website and Apiman docs.

Over time I'll flesh it out more as I get time to finish migrating over.

**Reply by msavy** [reply]
BTW, I notice the qmino docs are angularjs 1.x too. They don't seem to be actively developing it any more, though!

**Comment by spsvon** [comment]
Thanks team! You're awesome!
Sergio

**Reply by msavy** [reply]
I've migrated over a bunch more annotations and regenerated the docs with additional metadata.  Feel free to check out the website for the updated version.

It turned out to be a bit painful due to some glitchiness in the library I'm currently using, but hopefully that will be something I can make better in future.
